lib/views/bottom-status.js

```diff
@@ -9,24 +9,50 @@ class BottomStatus extends HTMLElement{
     this.iconSpan.classList.add('icon')
     this.appendChild(this.iconSpan)
 
-    this.count = 0
+    this._linting = false
+    this._count = 0
```

Comment:
We should keep only one codestyle, if we removed all the underscores it makes sense to removes these too.